blizzard.lua

```diff
@@ -98,5 +99,10 @@ function oUF:DisableBlizzard(unit)
 		-- Blizzard_ArenaUI should not be loaded
 		Arena_LoadUI = function() end
 		SetCVar('showArenaEnemyFrames', '0', 'SHOW_ARENA_ENEMY_FRAMES_TEXT')
+	elseif(unit:match('(nameplate)%d*$') == 'nameplate') then
```

Comment:
The only reason I did it was to preserve the style, take a look at ifs above this one :p

I honestly wanted to rewrite them all...